### ISSUE TITLE
Version 1 of react-select is no longer maintained

### ISSUE THREAD

**Issue by flexdinesh** [JedWatson/react-select#3981]
Version 1 of `react-select` is no longer supported.

In the best interest of the community we've decided to spend the time we have available on the latest versions. **We highly recommend upgrading from v1 to v2**.

If you're looking to fix bugs or improve `v1`, we recommend maitaining your own v1 fork. It was hard but necessary decision we had to make to sustain support for latest versions. We'll be closing the issues and PRs related to `v1` with the label `v1 deprecated`.

**How can I maintain my own v1 fork?**

[v1.x](https://github.com/JedWatson/react-select/tree/v1.x) is the branch where the latest v1 code resides. You can fork and pick up the code from that branch.

**How can I upgrade to latest versions?**

We have a [v1 to v2 upgrade guide](https://react-select.com/upgrade-guide) with detailed instructions.

_Note: The latest version is v3 but you don't have to follow a guide to upgrade from v2 to v3. You can just bump up your package version from v2._

**Relevant links**
- [v1 documentation and examples](v1.react-select.com)

**Comment by Slapbox** [comment]
We'd been hanging onto v1 for what seemed like forever and just moved to v3. I think even stragglers are probably upgrading by now.

**Comment by Lokua** [comment]
Sorry I know this is an old issue but we've been relying on the v1 version of react-select (specifically v1.3.0 which I assume was the last v1 release). The link above is a 404 and there are no releases or tags for it. We do not have capacity to upgrade to anything newer for some time and would like to fork it to add UNSAFE prefixes to lifecycle methods to remove the react warnings. Does the source for this exist anywhere?

I did try searching git logs for "v1.3.0" but nothing came up, so I manually went through the commit history to browse code and it appears the versions jump from v1.2.0 to v2.0.0-alpha.1 between [January 12 and 14, 2018](https://github.com/JedWatson/react-select/commits/master?after=eda88855b83a6ba528e557cc32da42926a993a63+2064&branch=master&qualified_name=refs%2Fheads%2Fmaster) (what happened on the 13th 😆 , I wonder).
